webapp/src/components/kanban/kanban.tsx

```diff
@@ -55,7 +56,19 @@ const vStrength = createVerticalStrength(Utils.isMobile() ? 60 : 250)
 
 const Kanban = (props: Props) => {
     const currentView = useAppSelector(getCurrentView)
+    const cardTemplates: Card[] = useAppSelector(getCurrentBoardTemplates)
     const {board, activeView, cards, groupByProperty, visibleGroups, hiddenGroups, hiddenCardsCount} = props
+    const [defaultTemplateID, setDefaultTemplateID] = useState('')
+
+    useEffect(() => {
+        if(currentView.fields && currentView.fields.defaultTemplateId) {
+            cardTemplates.forEach((cardTemplate) => {
+                if(cardTemplate.id === currentView.fields.defaultTemplateId) {
+                    setDefaultTemplateID(currentView.fields.defaultTemplateId)
+                }
+            })
```

Comment:
Suggest changing this to a `.find()`. Breaking out of a `.foreach` isn't possible, so you will continue to loop even after finding the expected value. (more important on a large set). `.find` will return the first element it finds and we only expect one. 

```suggestion
            if(cardTemplates.find(ct => ct.id === currentView.fields.defaultTemplateId)){
                setDefaultTemplateID(currentView.fields.defaultTemplateId)
            }
```

Reply:
Got it.